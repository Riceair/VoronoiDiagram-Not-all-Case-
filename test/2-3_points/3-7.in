#三點測試 x=4y 三點共線
3
4 1
100 25
400 100