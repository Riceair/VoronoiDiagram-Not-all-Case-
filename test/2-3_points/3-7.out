P 4 1
P 100 25
P 400 100
E -47000 189062.5 47500 -188937.5
E -15068 60493 15172 -60467
